tests/test_core.py

```diff
@@ -2323,6 +2323,13 @@ def test_pthread_equal(self):
   def test_pthread_dispatch_after_exit(self):
     self.do_run_in_out_file_test('pthread/test_pthread_dispatch_after_exit.c', interleaved_output=False)
 
+  @node_pthreads
+  def test_pthread_atexit(self):
+    # Test whether we can terminate a running thread during atexit.
```

Comment:
How about "Test that secondary threads are still running during atexit handlers"?

Reply:
I've rephrased it as "Test to ensure threads are still running when atexit-registered functions are called" with commit ea90c63dea2d1fbec29c4b727c09b6ed653b9306.